packages/cli/lib/commands/create.js

```diff
@@ -78,11 +78,16 @@ const options = [
 ];
 
 // Formulate Questions if `create` args are missing
-function requestParams(argv, templates) {
+function requestParams(name, argv, templates) {
 	const cwd = resolve(argv.cwd);
 
 	return [
 		// Required data
+		{
+			type: name ? null : 'text',
+			name: 'name',
+			message: 'The name of your application',
+		},
 		{
 			type: argv.template ? null : 'select',
```

Comment:
You could just swap this flag for `repo` from `create()` and the issue would be resolved.

Reply:
The intended way to supply additional arguments to a subcommand would be along with the flags that denote their purpose.
```diff
- preact create [template-name] [project-name]
+ preact create [name] --template [template-name]
```

```sh
preact create [name] [options]
```

It would be better to have a `--template` flag in that regard. Nevertheless, I would be happy to address the current issue if the proposed change is not intentional.

Reply:
Sure, but I don't think the template is an "additional" argument. It's required; we just also have a way to catch users if they forget it. 

Regardless, a major, as this currently is, will likely be a long ways away. Better to just fix up the behavior now.